src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -89,24 +93,34 @@ export class SkyFileAttachmentComponent implements AfterViewInit, AfterContentIn
     return this._disabled;
   }
 
-/**
- * Specifies the maximum size in bytes for valid files.
- */
+  /**
+   * Specifies the maximum size in bytes for valid files.
+   */
   @Input()
   public maxFileSize: number = 500000;
 
-/**
- * Specifies the minimum size in bytes for valid files.
- */
+  /**
+   * Specifies the minimum size in bytes for valid files.
+   */
   @Input()
   public minFileSize: number = 0;
 
+  /**
+   * Specifies a custom validation function. This validation is ran along with all internal
```

Comment:
```suggestion
   * Specifies a custom validation function. This validation runs alongside the internal
```

Reply:
Done - both in this component and the file drop component.